homework_6/package_1/Employee.java

```diff
@@ -0,0 +1,57 @@
+package package_1;
```

Comment:
I would ask you to remove redundant folders like .idea/, out/ from this pull request.
Moreover, you could prevent this from happening by adding `.gitignore` file to the root of your project and add respective patterns -> .idea/ out/ *.iml